app/controllers/providers_controller.rb

```diff
@@ -4,4 +4,8 @@ class ProvidersController < ApplicationController
   def index
     @providers = Provider.all
   end
+
+  def show
+    @provider = Provider.where(institution_code: params[:id]).first
```

Comment:
Provider.find(institution_code: param[:id])

?

Reply:
You can't use `find` when looking up anything other than the `id` and the `json_api_client` doesnt support `find_by`